airflow/www/static/js/dag.js

```diff
@@ -135,12 +135,18 @@ export function callModal(t, d, extraLinks, tryNumbers, sd, drID, mi) {
   mapIndex = mi;
   $('#dag_run_id').text(drID);
   $('#task_id').text(t);
-  $('#map_index').text(mapIndex);
   $('#execution_date').text(formatDateTime(d));
   $('#taskInstanceModal').modal({});
   $('#taskInstanceModal').css('margin-top', '0');
   $('#extra_links').prev('hr').hide();
   $('#extra_links').empty().hide();
+  if (mi >= 0) {
+    $('#modal_map_index').show();
+    $('#map_index, #modal_map_index .value').text(mi);
```

Comment:
Where is `#map_index`? I don't see that id anywhere

Reply:
Hmmm good question. This was already there, I just moved it. This might be setting the form control.

Reply:
I think `#` also matches `name` attributes. (Don’t take my words, I’m not fluent in js by any means.)

Reply:
It wasn't needed though, so I've removed this new addition, (left the second selector here)